docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+- (Optional) AWS CLI installed on your local machine. Refer to [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html). This is required if you plan to use EKSctl to create the EKS cluster and modify the IAM roles and policies locally.
+  
+### EKS Minimum Requirements
+
+{{< admonition type="caution" >}}
+These EKS requirements are the minimum specification needed to deploy Loki using this guide. You may wish to adjust plugins and instance types based on your AWS environment and workload. **If you choose to do so, this may invalidate the guide.**
+
+In this guide, we deploy Loki using `m5.xlarge` instances. This is a middle-of-the-road instance type that should work for most scenarios. However, you can modify the instance types and count based on your specific needs.
+{{< /admonition >}}
+
+The minimum requirements for deploying Loki on EKS are: 
+
+- Kubernetes version `1.30` or above.
+- `3` nodes for the EKS cluster.
+- Instance type depends on your workload. A good starting point is `m5.xlarge`.
+
+Here is the EKSctl cluster configuration file used in this guide:
+
+```yaml
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: loki
+  region: eu-west-2
+  version: "1.31"
+
+iam:
+  withOIDC: true
+
+addons:
+  - name: aws-ebs-csi-driver
+  - name: eks-pod-identity-agent
+
+managedNodeGroups:
+  - name: loki-workers
+    instanceType: m5.xlarge
+    desiredCapacity: 3
+    minSize: 2
+    maxSize: 3
+    amiFamily: AmazonLinux2
+    iam:
+      withAddonPolicies:
+        ebs: true
+    volumeSize: 80
+    volumeType: gp2
+    ebsOptimized: true
+
+```
+
+
+The following plugins are also required to be installed within the EKS cluster:
+- **Amazon EBS CSI Driver**: Enables Kubernetes to dynamically provision and manage EBS volumes as persistent storage for applications. We use this to provision the node volumes for Loki.
+- **Amazon EKS Pod Identity Agent**: Manages AWS IAM roles for pods, allowing fine-grained access control to AWS resources without needing to store credentials in containers. This is how Loki will access the S3 bucket.
+- **CoreDNS**: Provides internal DNS service for Kubernetes clusters, ensuring that services and pods can communicate with each other using DNS names. 
+- **kube-proxy**: Maintains network rules on nodes, enabling communication between pods and services within the cluster.
+
+You must also install an **OIDC (OpenID Connect) provider** on the EKS cluster. This is required for the IAM roles and policies to work correctly. If you are using EKSctl, you can install the OIDC provider using the following command:
+
+```bash
+eksctl utils associate-iam-oidc-provider --cluster loki --approve
+```
+**This may be installed by default when creating the EKS cluster using EKSctl and the above config.**
```

Comment:
Confused by this line.  Why is it bold?  Unclear on what "this" refers back to (OICD provider?).  Passive voice doesn't help either.  Are you trying to say that "The OICD provider was installed by default if you used the above config" ?